lib/engines/content_block_manager/features/edit_object.feature

```diff
@@ -2,16 +2,16 @@ Feature: Edit a content object
   Background:
     Given I am a GDS admin
     And the organisation "Ministry of Example" exists
-    And a schema "email_address" exists with the following fields:
+    And a schema "pension" exists with the following fields:
       | field         | type   | format | required |
-      | email_address | string | email  | true     |
-    And an email address content block has been created
+      | description   | string | string | true     |
```

Comment:
do we think it's ok to setup a pension without an embedded object of rate here? as the subschema functionality isn't under test here...
Maybe the confusion speaks to some tech debt around the tests here, that we are having to create bespoke ones each time we create a type and there could be a better organising of them?

Reply:
Yeah, I had a go at doing this, but I ended up getting tied up in knots